app/src/main/res/layout/act_calendar_editevent.xml

```diff
@@ -24,7 +24,9 @@
         android:layout_gravity="center"
         android:textSize="20sp"
         android:ems="10"
-        android:inputType="text"/>
+        android:inputType="text"
+        android:importantForAutofill="no"
```

Comment:
googled this, but didn't really get what it does, do you mind explaining me in the next meeting? :)

Reply:
I will explain it as well. But I will give it a shot here as well. 

This [link](https://developer.android.com/guide/topics/text/autofill-optimize) also explains it, but the way I got it is that the edittext framework/environment lets android determine wheter or not it is important for autofill, in the case it is it will give the user hints on what to put in. In order not to worry about that you can set the importantForAutofill to 'no' so that android will not worry about it. 
The inputtype is the kind of stuff that an edittext expects to get, based on that it can change the keyboard presented to the user. If I would say that the inputtype is double, the user would get a keyboard with only numbers. Now the user gets letters. More about that can be found [here](https://developer.android.com/training/keyboard-input/style).